### PR TITLE
ports "visualizes *cry emote" from tg

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

tgstation/tgstation#69492

except the bit where pepperspray causes the emote to trigger because I don't feel it properly expresses how much crying that would cause, instead cutting onions makes you cry

importantly IPCs and polys can't cry because they don't have visible eyes

# Spriting
crying animation 
image

# Changelog

:cl:  kawoppi
rscadd: cutting onions without eye protection will make you cry now
imageadd: crying now has an animation attached
/:cl:

### CODE REVIEW

**Review by cuackles — APPROVED** [review]
Yes

File: code/modules/hydroponics/grown/onion.dm [hunk 1]
```diff
@@ -28,6 +28,11 @@
 	slices_num = 2
 	wine_power = 30
 
+/obj/item/reagent_containers/food/snacks/grown/onion/slice(accuracy, obj/item/W, mob/user)
+	if(user && !user.is_eyes_covered())
```
Comment (redmoogle) [inline code/modules/hydroponics/grown/onion.dm]:
```suggestion
	if(isliving(user))
		var/mob/living/L = user
		if(!user.is_eyes_covered())
			user.emote("cry")
		. = ..()
```

**Review by redmoogle — CHANGES_REQUESTED** [review]
smh theos

**Comment by ToasterBiome** [comment]
now visible blushing

File: code/modules/hydroponics/grown/onion.dm [hunk 2]
```diff
@@ -28,6 +28,13 @@
 	slices_num = 2
 	wine_power = 30
 
+/obj/item/reagent_containers/food/snacks/grown/onion/slice(accuracy, obj/item/W, mob/user)
+	if(isliving(user))
+		var/mob/living/L = user
+		if(!user.is_eyes_covered())
```
Comment (redmoogle) [inline code/modules/hydroponics/grown/onion.dm]:
```suggestion
		if(!L.is_eyes_covered())
```

**Review by redmoogle — CHANGES_REQUESTED** [review]
whoops

**Review by redmoogle — APPROVED** [review]

**Pull request merged**